browser.js

```diff
@@ -17,7 +18,17 @@ module.exports = function(blob, cb){
           if (err) return cb(err);
 
           var buf = typedToBuffer(typed);
-          cb(null, parse(buf), buf);
+          var obj;
+
+          if (buf[0] === 60) {
+            obj = plistParse(buf.toString());
```

Comment:
Can you store those in a variable so we know without thinking which char this is?